src/helpers/getCourseInfo.js

```diff
@@ -0,0 +1,26 @@
+import { mockedAuthorsList } from '../constants';
+
+export function timeConvert(n) {
+	let num = n;
+	let hours = num / 60;
+	let rhours = Math.floor(hours);
+	let minutes = (hours - rhours) * 60;
+	let rminutes = Math.round(minutes);
+	return `${rhours} : ${rminutes} hours`;
+}
+
+export function dateConvert(date) {
+	return date.split('/').join('.');
+}
+
+export function getAutors(autors) {
+	let result = [];
```

Comment:
we can use there map instead of forEach, and new array will be automatically created

Reply:
done